http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -45,6 +47,10 @@ class CollectorService[F[_]: Sync](
   appVersion: String
 ) extends Service[F] {
 
+  // Contains an invisible pixel to return for `/i` requests.
+  val pixelBase64 = "R0lGODlhAQABAPAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=="
+  val pixel       = Stream.emit[F, String](pixelBase64).through(text.base64.decode)
```

Comment:
I think here you base64-decode the pixel for every request we receive, and the result is the same each time.  Would be nice to arrange the code so we have the pixel as an array of bytes when the collector first starts up.

For example, could use `java.util.Base64` to decode it when you first start up.  Or could store the raw pixel in the resources directory, and load it when you first start up.